packages/relay/src/lib/eth.ts

```diff
@@ -1017,8 +1017,17 @@ export class EthImpl implements Eth {
         }
       }
 
+      let cachedResponse = this.cache.get(`eth_call: ${call.data} from ${call.to}`);
```

Comment:
I'm concerned the `call.data` can get quite long.
In my exploration I went down the path of taking a hashing of the data to reduce it's length.

Maybe since the cache is so short lived it's not a biggie but let's make sure the key itself doesn't have length limits.
Maybe a simple approach of taking the first n characters also works if the hash approach is costly

Reply:
I just wanna avoid any potential error cases from a long data. maybe add a test case with a long data field if you stay with this approach.